resources/keywords/deploy_instance_dev.resource

```diff
@@ -16,6 +16,7 @@ User Selects A Project For Database Connection
     ...    AND    Set Project Namespace To ${newProject}
 
 Create New Project
+    User Closes Welcome Message
```

Comment:
Included "User Closes Welcome Message" keyword to close the welcome message pop up on the first time developer view. The Welcome message pops up a bit in delay which fails the execution.

Reply:
This keyword is already part of _User Selects ${perspective} View_ in _Login.resource_, any specific reason to include it here again?

Reply:
There is a situation for when the user is on developer view for the first time. The welcome message pops up when the user tries to perform any actions like creating a new project. To mitigate that issue, this keyword is been used again. 
This keyword is been used User Selects ${perspective} View already, but this one get skipped considering the DOM is ready, but when we try to perform any action, the pop up appears.

Reply:
@harshkumarRH I have updated the object properties, which seems to solve the issue